### ISSUE TITLE
Null effects of boot camps and short-format training for PhD students in life sciences

### ISSUE THREAD

**Issue by benmarwick** [carpentries/assessment-projects#28]
I saw this paper mentioned recently by @kariljordan in an email: http://www.pnas.org/content/114/37/9854 We should take a look at the methods it uses, see if we can use them on our data, and how our findings compare!

**Comment by kariljordan** [comment]
Yes! This paper has gotten a ton of discussion on Slack, and I believe @JasonJWilliamsNY has been talking with  [Rochelle Tractenberg](https://centile.georgetown.edu/Rochelle%20E.%20Tractenberg%2C%20PhD%2C%20MPH%2C%20PhD%2C%20PStat) about where it is correct and where it falls short. @karenword is drafting a [response](https://docs.google.com/document/d/1w_YBZESCHjijpOmaNXB5PAJQ-RXZlZOYlQCJn-r2T3o/edit?pli=1#) as well. I'm going to try my best to get some discussion going around this at the next [community call](http://pad.software-carpentry.org/community-call-2017-10-19). I'm so excited about this!

**Comment by karenword** [comment]
If Slack isn't your thing, a link to the (very messy at the moment) working document is here: https://github.com/carpentries/conversations/issues/13
